app/models/concerns/ldap_configs/ldap_import.rb

```diff
@@ -42,7 +42,11 @@ def process_entry entry
 
       state = if user
                 update_user user: user, data: data, roles: roles
-                user.roles.any? ? :updated : :deleted
+                if user.roles.any?
+                  user.saved_changes? ? :updated : :unchanged
+                else
+                  :deleted
```

Comment:
Acá es :deleted pero en el locale está como destroyed.

Reply:
_Mother of god_ ... cual preferimos dejar?

Me parece que este estado debería tocarlo nuevamente en la parte de `assign_managers`. Te parece poner un "intera-users" y re checkear el estado :updated/:unchanged ?

Reply:
(acabo de leer el otro comentario)

Reply:
Yo dejaría deleted, suena menos violento.

Mmm... me parece que dejaría el estado como está acá y en el manager "sobre-escribiría" SOLO los ```:unchanged``` si devuelve un ```saved_changes?```. En los otros casos no lo tocaría... solo por si se nos "pasa" algo... que al menos salga con un nombre =)